group-coordinator/src/test/java/org/apache/kafka/coordinator/group/assignor/OptimizedUniformAssignmentBuilderTest.java

```diff
@@ -294,26 +295,27 @@ public void testReassignmentForTwoMembersTwoTopicsGivenUnbalancedPrevAssignment(
             mkMapOfPartitionRacks(3)
         ));
 
-        Map<String, AssignmentMemberSpec> members = new TreeMap<>();
+        Map<String, MemberSubscriptionSpecImpl> members = new TreeMap<>();
+        Map<Uuid, Set<Integer>> targetAssignment;
 
-        members.put(memberA, new AssignmentMemberSpec(
-            Optional.empty(),
+        targetAssignment = mkOrderedAssignment(
+            mkTopicAssignment(topic1Uuid, 0, 1),
+            mkTopicAssignment(topic2Uuid, 0, 1)
+        );
+        members.put(memberA, new MemberSubscriptionSpecImpl(
             Optional.empty(),
             mkSet(topic1Uuid, topic2Uuid),
-            mkOrderedAssignment(
-                mkTopicAssignment(topic1Uuid, 0, 1),
-                mkTopicAssignment(topic2Uuid, 0, 1)
-            )
+            new Assignment(targetAssignment)
```

Comment:
Let's keep them inlined as they were. It avoids the local variable which is reused by different members.

Reply:
It's not very readable to me that way, but maybe we shouldn't use the same variable